### PR TITLE
Unit test: filtering OpenAI parameters

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Unit test: filtering OpenAI parameters

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Unit tests

### CODE REVIEW

File: tests/unit/llms/providers/test_openai.py [hunk 1]
```diff
@@ -26,7 +27,28 @@ def test_basic_interface():
...
+    """Test that now allowed parameters are removed before model init."""
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    # these two parameters should be removed before model init
+    params = {
+        "min_new_tokens": 1,
+        "max_new_tokens": 10,
+    }
+
+    openai = OpenAI(model="uber-model", params=params, provider_config=provider_config)
```
Comment (bparees) [inline tests/unit/llms/providers/test_openai.py:51]: where are you actually checking that the disallowed params were removed?

File: tests/unit/llms/providers/test_openai.py [hunk 2]
```diff
@@ -26,7 +27,28 @@ def test_basic_interface():
...
+    config.init_empty_config()  # needed for checking the config.dev_config.llm_params
+
+    openai = OpenAI(model="uber-model", params={}, provider_config=provider_config)
+    llm = openai.load()
+    assert isinstance(llm, ChatOpenAI)
+    assert openai.default_params
+
+
+def test_params_handling(provider_config):
+    """Test that now allowed parameters are removed before model init."""
```
[the remark this answers is not in the document]
Comment (tisnik) [reply tests/unit/llms/providers/test_openai.py]: fixed

File: tests/unit/llms/providers/test_openai.py (hunk 1 above)
Comment (tisnik) [reply tests/unit/llms/providers/test_openai.py:55]: right! added some checks

**Comment by onmete** [comment]
The proper parameters handling should be covered by https://issues.redhat.com/browse/OLS-363
but for now

/lgtm

**Pull request merged**